react/components/ProductAssemblyContext/Group.tsx

```diff
@@ -62,15 +62,13 @@ export const ProductAssemblyGroupContextProvider: FC<ProductAssemblyGroupContext
     return acc
   }, {})
 
-  const initialState: AssemblyOptionGroupState = {
-    ...assemblyOption,
-    path: groupPath,
-    quantitySum,
-    optin: assemblyOption.required,
-    valuesOfInputValues,
-  }
+  assemblyOption.path = assemblyOption.path ?? groupPath
```

Comment:
can't we try with the initializer solution?

Reply:
Change to use the initializer solution